flags.go

```diff
@@ -101,13 +101,15 @@ func defineFlagVar() *FlagVar {
 		"indicates the current log-level, enter negative values to increase verbosity (e.g. 9)",
 	)
 	flag.BoolVar(&flagVar.inKCPMode, "in-kcp-mode", false,
-		"indicates lifecycle manager is deployed in control-plane mode")
+		"Indicates lifecycle manager is deployed in control-plane mode")
 	flag.BoolVar(&flagVar.enablePurgeFinalizer, "enable-purge-finalizer", false,
 		"Enabling purge finalizer")
 	flag.DurationVar(&flagVar.purgeFinalizerTimeout, "purge-finalizer-timeout", defaultPurgeFinalizerTimeout,
 		"Indicates the SKR Purge Finalizers execution delay in seconds")
 	flag.StringVar(&flagVar.skipPurgingFor, "skip-finalizer-purging-for", "", "Exclude the passed CRDs"+
 		" from finalizer removal. Example: 'ingressroutetcps.traefik.containo.us,*.helm.cattle.io'.")
+	flag.StringVar(&flagVar.remoteSyncNamespace, "sync-namespace", "kyma-system",
```

Comment:
it's better to extract this as a const, and it can be provided in the test as input value for testing.

Reply:
Done.